helm/superset/values.yaml

```diff
@@ -248,6 +248,8 @@ supersetNode:
     maxReplicas: 100
     targetCPUUtilizationPercentage: 80
     # targetMemoryUtilizationPercentage: 80
+  # -- Sets the [pod disruption budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) for supersetNode pods
```

Comment:
Can you add an `enabled` field here, along with a set of sensible defaults for each of these blocks? That way, we can add the various settings to the schema and can document them appropriately. Using an `enabled` also makes the config a bit more explicit

e.g.
```yaml
podDisruptionBudget:
  minAvailable: 1
  maxUnavailable: 1
```

Reply:
Thanks, I will take this into account. However both variables are mutually exclusive so we would have to comment these in `values.yaml` and document somehow that PDB only accepts either `minAvailable` or `maxUnavailable` at a time - https://kubernetes.io/docs/tasks/run-application/configure-pdb/#specifying-a-poddisruptionbudget.

Reply:
Yep, that makes sense. For extra credit, you could fail if both are set:

```helm
{{- if .Values.podDisruptionBudget.minAvailable and .Values.podDisruptionBudget.maxAvailable }}
{{ required "Only one of minAvailable or maxAvailable should be set" .Values.podDisruptionBudget.minAvailable }}
{{- end}}
```

Reply:
I've made the changes and slightly refactored the PDB templates I'm contributing there to make better use of variable scoping - reducing a bit the boilerplate and avoiding to repeat `.Values.supersetXXX.podDisruptionBudget` everywhere.

Please tell me if the updated documentation looks clear enough as well. Thanks!